### ISSUE TITLE
Exporting backups doesn't work, just makes a backup of the original?

### ISSUE THREAD

**Issue by Nawor3565** [kwsch/PKHeX#707]
Okay, so I wanted to change a bunch of stuff about my Pokemon, mainly things like nicknames for the ones I got in trades, and a ton of other things. So I exported my save file, put it in PKHex, and changed everything I wanted to. I decided I wanted to try out the new stuff in batch editor, so I made a backup first by going File -> Export Save -> Create Backup. I had also made a few backups when I was changing things before. Long story short, I changed some things I shouldn't have, but no problem, I'll just load a backup. Nope.
All of the backups I made were just copies of my save file when I first loaded it into PKHex. Meaning about an hour and a half of work was gone. I'm not sure what the problem is here, is this intended? I doubt it, but if it is, could you make it more obvious what creating a backup means so this doesn't happen to anyone else?

TL;DR: Imported my save file and changed a bunch of stuff. Made a backup .bak before a bulk edit, messed stuff up, and tried to load my backup. Backup was the same as when I first imported my save, and over an hour of work was lost.

**Comment by kwsch** [comment]
A backup is an unmodified file (ie the original file loaded).
You can export a {current file} by exporting the (modified) main save.

**Comment by Nawor3565** [comment]
Makes sense. But, because of how it's worded, it seems (to me, at least) like it would export a backup file, not export a backup that it made originally. Is there any way you could re-word it to make it's function more clear?

**Comment by kwsch** [comment]
Via [Wikipedia](https://en.wikipedia.org/wiki/Backup):

>In information technology, a backup, or the process of backing up, refers to the copying and archiving of computer data so it may be used to **restore the original** after a data loss event.

PKHeX stores two states of the save file:
1. The original, unmodified save file (exportable via backup).
2. The current, modified save file (exportable via export main).

You chose the wrong export option; I don't think changing the naming would clear things up as it's already described succinctly.

**Comment by Nawor3565** [comment]
Ah, I think there was a bit of confusion. I thought that it meant "make a backup," not "export the backup that was already made." I guess there's no real way to word it differently, it's just how it's interpreted.
